eip-draft_babyjubjub.md

```diff
@@ -0,0 +1,362 @@
+---
+eip:      2494
+title:    Baby Jubjub Elliptic Curve
+authors:  Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
+discussions-to: https://github.com/ethereum/EIPs/pull/2494
```

Comment:
This should link to `https://github.com/ethereum/EIPs/issues/2493`

Reply:
Issue #2493 is closed. The discussion thread is moved to https://ethresear.ch/t/discussions-to-thread-for-eip-2494-baby-jubjub-elliptic-curve/6845.

Reply:
FYI: At the time I wrote the comment, the issue was still open.

Reply:
I opened and closed it now, could you check if it is okay now?

Reply:
Hm? I wrote the comment at 10:16 GMT and the issue was *first* closed 10:45 GMT. I've only mentioned it for the record.